great_expectations/validation_operators/validation_operators.py

```diff
@@ -19,6 +19,42 @@
 # NOTE: Abe 2019/08/24 : This is first implementation of all these classes. Consider them UNSTABLE for now.
 
 
+class ValidationOperatorResult(object):
+    # TODO this is a design sketch
+    def __init__(self):
+        raise NotImplementedError
+
+    def get_run_id(self):
+        raise NotImplementedError
+
+    def list_assets_validated(self) -> list:
+        raise NotImplementedError
+
+    def get_evaluation_parameters(self):
+        raise NotImplementedError
+
+    @property
+    def statistics(self) -> dict:
+        raise NotImplementedError
+
+    @property
+    def actions_results(self) -> dict:
+        raise NotImplementedError
+
+    @property
+    def success(self) -> bool:
+        raise NotImplementedError
+
+    @property
+    def details(self):
```

Comment:
Given pulling of validation results and actions results out of details...I'd say this is just a generic dictionary at this point that is there mostly in case another operator needs it (analogous to expectationvalidationresult)